code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -79,7 +79,15 @@
 	if(GS)
 		qdel(GS)
 		eaten = TRUE
-
+	
+	var/mob/living/carbon/human/species/pod/PP = locate(/mob/living/carbon/human/species/pod) in loc

```

Comment:
that check doesn't work, it's a special adminspawn subtype that sets the human to podperson instead of having to do it manually, but ingame there are only mob/living/carbon/human

Reply:
>:( well how would i do it then

Reply:
You could have it check for a nearby human, do an istype check to see if the mob's species is set to that of a podperson, and then have it do the thing you want it to do.

Reply:
Also this only works for mobs on the same tile as pete

Reply:
has someone made an IsSpecies proc yet

Reply:
also you should use oview